src/components/ArticlePage.jsx

```diff
@@ -23,7 +27,21 @@ const ArticlePage = () => {
     });
   }, [article_id]);
 
-  console.log(comments[0]);
+  const handleCroak = (value) => {
+    if (value === "up") {
+      setLoadingUp(true);
+      patchArticleVotes(article_id, 1).then((res) => {
+        setVotes(votes + 1);
+        setLoadingUp(false);
+      });
+    } else {
+      setLoadingDown(true);
+      patchArticleVotes(article_id, -1).then((res) => {
+        setVotes(votes - 1);
+        setLoadingDown(false);
+      });
```

Comment:
What happens if something goes wrong? This is part of a later ticket too, but a catch here could be useful, then you could reset the votes if there's an error